addon/mixins/registration-actions.js

```diff
@@ -0,0 +1,68 @@
+
+import Ember from 'ember';
+
+export default Ember.Mixin.create({
+    node: null,
+    draft: null,
+    model: null,
+    _node: Ember.computed.or('node'),
+    _draft: Ember.computed.or('draft', 'model'),
+    /** Updates current registration metadata with new responses to questions.
+     **/
+    _updateMetadata(d, u) {
```

Comment:
Please avoid single-character variable names (except _maybe_ in single-line anonymous functions)